magic-gds-drc/README.src.rst

```diff
@@ -0,0 +1,24 @@
+``skywater-pdk-actions`` - ``magic-gds-drc``
```

Comment:
`check-drcs-for-cell-gds-using-magic` would be a better name.

The most important information should be earlier. Your description is pretty good;
> This GitHub action runs Design Rule Checks on all GDS files inside the /cells directory.

`This GitHub action runs` --> Well it's in a repo called `skywater-pdk-actions`.
`Design Rule Checks` --> `drc-`
`on all GDS files inside the /cells directory.` --> `for-cell-gds`
`These checks are done with the Magic tool.` --> `using-magic`.

`uses: docker://ghcr.io/donn/skywater-pdk-actions-check-drcs-for-cell-gds-using-magic:latest`

`s/donn/google/`?